requirements.txt

```diff
@@ -2,3 +2,4 @@ six
 pexpect
 pyparsing
 pynml
+-e git+git://github.com:HPENetworking/pyszn@1.0.0#egg=pyszn
```

Comment:
Unlike the requirements.dev.txt which is used for testing (tox - pip), this will not work here, as the `install_requires` attribute of the `setup()` function (setup.py) expects a list of strings and I think it does not support the `-e` option of pip.
You'll have to release pyszn to pypi, or document the how to get this non pypi requirement manually.

Reply:
Thanks for noting this is here, will fix once we have a agreed on away to handle environment passing to the platform engines.